routes.js

```diff
@@ -56,13 +56,29 @@ router.post("/postWorkerDetail", async (req, res) => {
   });
   try {
     const workerSave = await workerPost.save();
+    const dataKeys = Object.keys(req.body);
+    console.log("Data keys:", dataKeys);
     res.json(workerSave);
   } catch (error) {
     res.send("error");
     console.log(error);
   }
 });
 
+// Route to get all unique keys in the worker collection
+router.get("/workerKeys", async (req, res) => {
+  try {
+    const workers = await Worker.find();
+    const keys = [
+      ...new Set(workers.flatMap((worker) => Object.keys(worker.toObject()))),
```

Comment:
It's not the right way to get keys, you have to use mongoose query only instead of using  js map function